views/configure.pug

```diff
@@ -26,5 +26,46 @@ block content
       label(for='confirm-password').control-label Confirm password
       input(type='password',name='confirm-password',value='',required=true,data-match='#password',data-match-error="The password and the confirmation must match").form-control#confirm-password
       span.help-block.with-errors
+    div.consent_conf
+      +consent_form
+      div.form-group
+          div.checkbox
+            label.control-label
+              input(type='checkbox',name='agree_consent',id='register-agree-consent',value='1',required=true)
+              =_("I agree to grant consent to this research.")
     div.form-group
       button(type='submit').btn.btn-primary Configure
+    
+
+
+mixin consent_form
+  h2=_("Consent Form")
+
+  div.consent-form
+    p
+      | You are invited to participate in a research study on virtual assistants. With your consent, the Genie virtual assistant will record the commands you give it.
+      | Recording the commands will allow Genie to improve its understanding of natural language. The collection is completely confidential, and personal data (such as emails and phone numbers) will be stripped automatically.
+      | You must be 18 or older to participate, and currently residing in the United States.
+
+    h3 Risks and Benefits
+    p
+     | We do not foresee any risk from this study. The benefits which may reasonably be expected to result from this study are that the open-source Genie virtual assistant improves over time. We cannot and do not guarantee or promise that you will receive any benefits from this study.
+     //-| Your decision whether or not to participate in this study will not affect the functionality of the Genie service, or your ability to use it.
```

Comment:
This line should be uncommented for almond-server

Reply:
fixed